package.json

```diff
@@ -38,24 +48,31 @@
     "mongoose": "5.3.12",
     "node-schedule": "1.3.0",
     "web3": "1.0.0-beta.36",
+    "web3-eth-contract": "^1.0.0-beta.55",
```

Comment:
Please target all dependencies with strict versions. That's the convention we currently have.